odoo/addons/base/wizard/base_module_uninstall_views.xml

```diff
@@ -16,27 +16,24 @@
                         <div class="me-auto p-2 bd-highlight"><h3>Apps to Uninstall</h3></div>
                         <div class="p-2 bd-highlight"><field name="show_all"/> Show All</div>
                     </div>
-                    <field name="module_ids" mode="kanban" class="o_modules_field">
+                    <field name="module_ids" mode="kanban">
```

Comment:
either o_modules_kanban is no more usefull or o_modules_field is needed

Reply:
@omra-odoo?

Reply:
i think both the classes is needed, otherwise this [css](https://github.com/odoo/odoo/blob/2265cec7616de2d660fcd76718142c1d77d7a0ff/odoo/addons/base/static/src/css/modules.css#L40) won't be applied

Reply:
do we still need that style?